kahuna/public/js/components/gr-collections-panel/gr-collections-panel.html

```diff
@@ -1,5 +1,4 @@
-<div class="panel panel__left panel--side"
-     ng:class="{ 'panel__left--hidden': !ctrl.isVisible }">
+<div>
```

Comment:
Are you removing the `panel` class on purpose? Seems to leave the other `panel__` classes below dangling without a parent.